yarn.lock

```diff
@@ -54,10 +54,10 @@
     json-stringify-deterministic "^1.0.8"
     multiformats "^9.6.4"
 
-"@cowprotocol/permit-utils@0.0.1-RC.1":
-  version "0.0.1-RC.1"
-  resolved "https://registry.yarnpkg.com/@cowprotocol/permit-utils/-/permit-utils-0.0.1-RC.1.tgz#3c0f24ebc735fdf02d763c8c840ff2ccbd688a74"
-  integrity sha512-9Sdm5oaNJiQpcYUTYUVkQtRyuflsHFlWXWdHh9uBnMWTf6MnEGciAqZPNrenGKYI7sw0dIIEPdoB0L6HbdVfQw==
+"@cowprotocol/permit-utils@0.0.1-RC.2":
+  version "0.0.1-RC.2"
+  resolved "http://localhost:4873/@cowprotocol/permit-utils/-/permit-utils-0.0.1-RC.2.tgz#9d3a07b8aa73d3683e5e562b512be93f092ebc8c"
+  integrity sha512-gWjrtI+JzNvAma5cC3zP7y9dseTY1xupFRM9rDexUa6VZmczMy5oXEBC4NVloyYtvSNd5v1rBoyuVJngOXCTEA==
```

Comment:
This obviously won't work on CI.
Will update once https://github.com/cowprotocol/cowswap/pull/3374 is published

Reply:
okok